bundles/org.openhab.binding.broadlink/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -46,23 +46,39 @@
 		<item-type>Switch</item-type>
 		<label>Night Light</label>
 		<category>Lightbulb</category>
+		<tags>
+			<tag>Switch</tag>
+			<tag>Light</tag>
+		</tags>
 	</channel-type>
 	<channel-type id="light">
 		<item-type>String</item-type>
 		<label>Illumination</label>
 		<description>Light conditions</description>
+		<tags>
+			<tag>Status</tag>
+			<tag>Illuminance</tag>
+		</tags>
 		<state readOnly="true"></state>
 	</channel-type>
 	<channel-type id="noise">
 		<item-type>String</item-type>
 		<label>Background Noise</label>
 		<description>Noise conditions</description>
+		<tags>
+			<tag>Status</tag>
+			<tag>Noise</tag>
+		</tags>
 		<state readOnly="true"></state>
 	</channel-type>
 	<channel-type id="air">
 		<item-type>String</item-type>
 		<label>Air Quality</label>
 		<description>Air quality conditions</description>
+		<tags>
+			<tag>Status</tag>
```

Comment:
```suggestion
			<tag>Measurement</tag>
```

Reply:
.. but thinking about this again.. Generally I would tend to use Measurement on numeric values. This allows (say) the UI badges to calculate (say) min, max or averages. And using Measurement on Strings might break that. I am not completely sure. That is why I proposes Status instead. => WDYT?

Reply:
Ah i see the struggle. Can't provide you with a clear definition, but i agree that a String value like 'good', 'bad' translates better to `Status` then to `Measurement` so let's keep it like it is.

Reply:
I will revert.